lib/std/http/Server.zig

```diff
@@ -303,6 +303,51 @@ pub const Request = struct {
         return http.HeaderIterator.init(r.server.read_buffer[0..r.head_end]);
     }
 
+    /// Retrieves the value of a specified HTTP header from a Request object.
```

Comment:
This should mention that in the case of duplicate headers the first one is being returned. This is not necessarily in line with RFC 2616 which states that duplicate headers are equivalent to a single comma-separated value. (`Foo: a\nFoo: b` = `Foo: a, b`)

Reply:
This is great feedback, thank you. Adjusting the comment to reflect behavior.